Automation/create_vm_script.ps1

```diff
@@ -0,0 +1,68 @@
+#Script to create a Virtual Machine on Azure.
+#https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-powershell
+
+#Note: This creates a Windows VM, but the end goal is a Linux VM. This will be the next sprint.
+#Note: There are extra parameters that are not being used yet. These will be used when creating Linux machine.
+param (
+    [string] [Parameter(Mandatory=$true)] $tenantId, #Tenant ID used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory=$true)] $applicationId, #App ID used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory=$true)] $secret, #Azure Secret used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory=$true)] $subscriptionId, #Subscr. ID used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory)] $rgname, #Resource Group Name required.
+    [string] [Parameter(Mandatory)] $vmname, #VM Name required
+    [string] $location='West US2', #Default value for RG/VM Location
+    [string] $vnetname='myVnet',  #Default value
+    [string] $subnetname='mySubnet', #Default value
+    [string] $nsgname='myNetworkSecurityGroup', #Default value
+    [string] $publicipname='myPublicIpAddress', #Default value
+    [string] $dnsLabelPrefix=$rgname #Default value
```

Comment:
I think this is a good temporary solution - I don't know if we'll eventually want those to be input parameters, but it will be really easy for you to change that if required, since you define them here. : )